front/src/pages/nft/[id].tsx

```diff
@@ -31,7 +31,7 @@ const SingleNFTPage = (props: SingleNFTPageProps) => {
             <GCContainer>
                 <GCGrid className={classes.grid}>
                     <GCGridCol sm={6} md={6}>
-                        <GCImage src={data.nftImageUrl} alt="gun" />
+                        <GCImage src={data.nftImageUrl} alt="gun"/>
                     </GCGridCol>
                     <GCGridCol sm={6} md={6}>
                         <NftCardInformation data={data}></NftCardInformation>
```

Comment:
please dont use empty tags, close them in the same place you opened it, it leads to warnings